indico/migrations/versions/20210211_1613_26985db8ed12_add_attach_ical.py

```diff
@@ -0,0 +1,25 @@
+"""add_attach_ical
```

Comment:
```suggestion
"""Add attach_ical to reminders
```

Tip: The `-m` when generating the revision can be more human-friendly, it's automatically slugified for the filename.